dolphinscheduler-ui/src/views/resource/file/index.tsx

```diff
@@ -15,343 +15,16 @@
  * limitations under the License.
  */
 
-import { useRouter } from 'vue-router'
-import { defineComponent, onMounted, ref, reactive, Ref, getCurrentInstance } from 'vue'
-import {
-  NIcon,
-  NSpace,
-  NDataTable,
-  NButtonGroup,
-  NButton,
-  NPagination,
-  NInput,
-  NBreadcrumb,
-  NBreadcrumbItem
-} from 'naive-ui'
-import { useI18n } from 'vue-i18n'
-import { SearchOutlined } from '@vicons/antd'
-import { useTable } from './table/use-table'
-import { useFileState } from './use-file'
-import { BreadcrumbItem, IRenameFile } from './types'
-import { useFileStore } from '@/store/file/file'
-import {
-  queryCurrentResourceById,
-  queryResourceById
-} from '@/service/modules/resources'
-import Card from '@/components/card'
-import ResourceFolderModal from './folder'
-import ResourceUploadModal from './upload'
-import ResourceRenameModal from './rename'
-import styles from './index.module.scss'
-import type { ResourceFile } from '@/service/modules/resources/types'
-import type { Router } from 'vue-router'
+import { defineComponent } from 'vue'
+import ResourceListModal from '../components/resource'
 
 export default defineComponent({
-  name: 'File',
+  name: 'FILE',
```

Comment:
It seems that File is better than FILE for name.

Reply:
resolved